packages/sitecore-jss/src/editing/graphql-editing-service.ts

```diff
@@ -118,13 +118,13 @@ export class GraphQLEditingService {
   async fetchEditingData({
```

Comment:
Might need to add a unit test, you can name it like: "should fetch editing data with missing optional parameters"